desktop/frontend/package.json

```diff
@@ -34,6 +35,7 @@
     "react-dom": "18.2.0",
     "react-draggable": "^4.4.5",
     "react-markdown": "^8.0.3",
+    "react-syntax-highlighter": "^15.5.0",
```

Comment:
Seems `react-syntax-highlighter` is a replacer of `rehype-highlight` below?

Reply:
The effect of rehipe height is not very good

Reply:
So, if we dropped usage of `rehype-highlight`,as well as delete this requirements?

Reply:
I think we can remove rehype-highlight

Reply:
Only the MarkDown component is used and can be replaced with one